dev: re-enable auto-formatting of `init-[moves|abilities].ts`

DayKev opened the pull request:
## What are the changes the user will see?
N/A

## Why am I making these changes?
Move towards entirely removing ESLint.

## What are the changes from a developer perspective?
Automatic formatting re-enabled for `initAbilities()` and `initMoves()` by adding `//` to the end of each `new [*Move|Ability]()` line to force specific formatting.

## Checklist
- [x] Otherwise: **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?

DayKev committed:
dev: re-enable auto-formatting of `init-[moves|abilities].ts`

The special ESLint formatting rules for these files are now deleted

MokaStitcher commented on src/data/init/init-moves.ts:
```diff
@@ -982,229 +977,230 @@ export function initMoves() {
         user.hasTag(BattlerTagType.UPROAR)
-          // "{pokemonNameWithAffix} is making an uproar!"
-          ? i18next.t("moveTriggers:isMakingAnUproar", { pokemonNameWithAffix: getPokemonNameWithAffix(user) })
+          ? // "{pokemonNameWithAffix} is making an uproar!"
```
That's weird xd

DayKev replied:
Yeah, it's slightly annoying but 🤷.

MokaStitcher commented on src/data/init/init-moves.ts:
```diff
@@ -1859,315 +1852,312 @@ export function initMoves() {
-            return (lastPlayerFaint !== undefined && turn - lastPlayerFaint.turn === 1) ? 2 : 1;
-          }
+        if (user.isPlayer()) {
+          return lastPlayerFaint !== undefined && turn - lastPlayerFaint.turn === 1 ? 2 : 1;
```
I feel it was easier to read with the parentheses

DayKev replied:
The condition could be pulled out (e.g. `const condition = ...; return condition ? a : b;`). Arguably that'd be better than both.

DayKev replied:
Formatted it in a way that should be more readable now.

MokaStitcher commented on src/data/init/init-moves.ts:
```diff
@@ -1859,315 +1852,312 @@ export function initMoves() {
-        },
-      ),
-    new AttackMove(MoveId.FINAL_GAMBIT, ElementalType.FIGHTING, MoveCategory.SPECIAL, -1, 100, 5, -1, 0, 5)
+        return lastEnemyFaint !== undefined && turn - lastEnemyFaint.turn === 1 ? 2 : 1;
```
Same. If it's done automatically by biome then don't bother I suppose 🤷

MokaStitcher commented on src/data/init/init-moves.ts:
```diff
@@ -2373,236 +2361,249 @@ export function initMoves() {
+    new AttackMove(MoveId.BREAKNECK_BLITZ__SPECIAL, ElementalType.NORMAL, MoveCategory.SPECIAL, -1, -1, 1, -1, 0, 7) //
       .unimplemented(),
-    new AttackMove(MoveId.ALL_OUT_PUMMELING__PHYSICAL, ElementalType.FIGHTING, MoveCategory.PHYSICAL, -1, -1, 1, -1, 0, 7)
+    new AttackMove(
```
Seems like the // didn't work out for this one, because it's too long?

DayKev replied:
Oh I missed that, whoops.

MokaStitcher commented on src/data/init/init-moves.ts:
```diff
@@ -2373,236 +2361,249 @@ export function initMoves() {
+    new AttackMove(MoveId.SAVAGE_SPIN_OUT__SPECIAL, ElementalType.BUG, MoveCategory.SPECIAL, -1, -1, 1, -1, 0, 7) //
       .unimplemented(),
-    new AttackMove(MoveId.NEVER_ENDING_NIGHTMARE__PHYSICAL, ElementalType.GHOST, MoveCategory.PHYSICAL, -1, -1, 1, -1, 0, 7)
+    // biome-ignore format: just barely too long
```
Does the next line still need // since we have this?

MokaStitcher commented on src/data/init/init-moves.ts:
```diff
@@ -2373,236 +2361,249 @@ export function initMoves() {
+    new AttackMove(MoveId.NEVER_ENDING_NIGHTMARE__PHYSICAL, ElementalType.GHOST, MoveCategory.PHYSICAL, -1, -1, 1, -1, 0, 7) //
       .unimplemented(),
-    new AttackMove(MoveId.NEVER_ENDING_NIGHTMARE__SPECIAL, ElementalType.GHOST, MoveCategory.SPECIAL, -1, -1, 1, -1, 0, 7)
+    // biome-ignore format: just barely too long
```
Same

MokaStitcher commented on src/data/init/init-moves.ts:
```diff
@@ -2612,114 +2613,112 @@ export function initMoves() {
       .attr(FormChangeItemTypeAttr),
     // #region Pikachu signature Z-Move (unused)
-    new AttackMove(MoveId.TEN_MILLION_VOLT_THUNDERBOLT, ElementalType.ELECTRIC, MoveCategory.SPECIAL, 195, -1, 1, -1, 0, 7)
+    // biome-ignore format: just barely too long
```
Same

MokaStitcher commented on src/data/init/init-moves.ts:
```diff
@@ -3352,187 +3343,182 @@ export function initMoves() {
+    new AttackMove(MoveId.ELECTRO_DRIFT, ElementalType.ELECTRIC, MoveCategory.SPECIAL, 100, 100, 5, -1, 0, 9) //
       .attr(MovePowerMultiplierAttr, (user, target, move) =>
-        target.getAttackTypeEffectiveness(move.type, user) >= 2 ? 5461 / 4096 : 1,
+        target.getAttackTypeEffectiveness(move.type, user) >= 2 ? 1.33 : 1,
```
Is this really the PR to chance this? (I know others in the team have feelings about being exact to jow mainline computes things, we might want to run the change past them)

DayKev replied:
Made it a separate PR.

MokaStitcher reviewed (commented):
lgtm
Should we add a comment at the top of the files to explain what the // do?

DayKev committed:
Make `MovePowerMultiplierAttr` definition more readable

Fix formatting of unused `MoveId.ALL_OUT_PUMMELING__PHYSICAL`
and remove some unnecessary `//` due to suppression comments

Revert `/ 4096` removal

DayKev committed:
Merge branch 'beta' into dev/enable-auto-formatting-of-init-files

DayKev committed:
Put `TODO` comment back

innerthunder committed:
Merge branch 'beta' into dev/enable-auto-formatting-of-init-files
